tools/scripts/db-update.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
 cd libs/java/dal/db
-docker build -t tno:db-migration dal-db-migration --no-cache --force-rm
+docker build tno:db-migration dal-db-migration --no-cache --force-rm
```

Comment:
This won't work.  Need the `-t`

Reply:
Added it back in again. This is another last-minute change that wasn't saved.

Reply:
I don't see it?

Reply:
It was -i now it's -it?